### ISSUE TITLE
Install package in Windows Phone solution

### ISSUE THREAD

**Issue by gsferreira** [dillenmeister/Trello.NET#25]
Has anyone installed the Nuget package successively in a WP 7.1 solution? Is it possible?

Thanks

**Comment by dillenmeister** [comment]
No effort has been put in to get it to work on WP, so if it does it's just "luck".

Did you try? What happened?

**Comment by gsferreira** [comment]
The RestSharp implementation for Portable Class Libraries doesn't support synchronous requests like this:
`var response = Execute<T>(request);`

**Comment by dillenmeister** [comment]
Do the methods on Trello.Async work?

**Comment by gsferreira** [comment]
Yes. Portable RestSharp doesn't implement Execute but ExecuteAsync is there.

**Comment by dillenmeister** [comment]
Ok, so you can use Trello.NET in a WP project then?

**Comment by gsferreira** [comment]
No, because you can't compile it for a Portable version.
because in trello.net we have Execute that does not exists in restsharp Portable

**Comment by dillenmeister** [comment]
Do you have any ideas how to make Trello.NET work on WP? What alternatives are there?

**Comment by gsferreira** [comment]
@dillenmeister I've made a fork of Trello.NET to make it compatible with Portable Libraries.
You can see here https://github.com/gsferreira/Trello.NET the changes that I've made. I didn't test every methods.
You can also take a look into RestSharp repository to get some inspiration https://github.com/restsharp/RestSharp
